docusaurus/docs/core-concepts/clients/tunnelers/04-macos.md

```diff
@@ -1,3 +1,48 @@
 # MacOS
 
-The MacOS tunneler is currently in preview. Find the app in the [Mac App Store](https://apps.apple.com/us/app/ziti-tunnel/id1460484572?mt=12)
+## Installation

```

Comment:
I don't think we need to cover precisely how to install an app in macOS, just provide the link to "Ziti Desktop Edge".

Reply:
removed